src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
```suggestion
        Machine truck = new Truck();
```
ми можемо використати абстрактний тип для посилання, хоча всі посилання будуть типу Machine методи викличуться різні через поліморфізм

Reply:
Якщо я присвоїв так посилання до обєкту -  Machine truck = new Truck ();
Якщо у класі Truck є метод якого немає в Machine. Я не можу його викликати правильно?
Наприклад метод truck.buzzerOn();

Reply:
так, саме посилання вирішує, що за методи можуть бути викликані а об'єкт, що лежить за посиланням дає імплементацію для цих методів

Reply:
Дякую